packages/x-components/src/x-modules/experience-controls/store/types.ts

```diff
@@ -0,0 +1,52 @@
+import { XStoreModule } from '../../../store/store.types';
+import { ExperienceControlsConfig } from '../config.types';
+
+/**
+ * Experience Controls store state.
+ *
+ * @public
+ */
+export interface ExperienceControlsState {
+  /**
+   * Configuration for the `ExperienceControls` module.
+   */
+  config: ExperienceControlsConfig;
+}
+
+/**
+ * Experience Controls store getters.
+ *
+ * @public
+ */
+export interface ExperienceControlsGetters {
+  getNumberCarousels(state: any): number;
+  getResultsCarousels(state: any): number;
```

Comment:
This getters don't make much sense. The getters are usually used to retrieve a property of the state with a manipulation. For example a list that is stored in the state and we want to retrieve it sorted. We would create a getter called `sortedList` that will take the list from the state, sort it and return it.

If we want to retrieve the property of the state as is, we take it directly from the state (usually).